### ISSUE TITLE
Add. 12,163, foll. 127–304 | https://bl.vuexistapps.us/manuscript/42 (Part2)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#698]
### Problems with Manuscript Overview

- On the web page, this manuscript (Add. 12,163, foll. 127–304) is divided into two parts. So, in total, there are three parts, the first of which consists of the last two. I am not aware of the Syriaca guidelines for such cases, but perhaps, in terms of nesting, parts 3 & 4 should be nested into part 2. 
- Wright's Subject Classification is empty. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

- Lack of space in "consistingof"
- The sentence "The former seems to have been transcribed by at least two persons (foll. 128 b—197 and foll. 198—224). Both are written in current hands of the xthor xith cent." is included in the main paragraph of Phys. Des. and in a slightly different form, it also appears in the Hands section. 
- Hand 1: Lack of space in "Seeadditions"
- Hand 2: Lack of space in "Seeadditions"
- Hand 3: Lack of space in "Seeadditions"
- Hand 4: Lack of space in "Seeadditions"
- Hand 5: Lack of space in "Seeadditions"
- Hand 6: Lack of space in "Foll.128b-197seems"
- Hand 6: Lack of space in "thexth"
- Hand 7: Lack of space in "Foll.198-224seems"
- Hand 7: Lack of space in "thexth"
- Hand 8: Lack of space in "Foll.225-304seems"
- Hand 8: Lack of space in "thexth"
- Minor hands (1-5) are mentioned first, while major hands (6-8) are placed at the end. 
- Accompanying Material is empty. 

### Categories of Physical Description Problems

- [X] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
@wlpotter 
I feel like #42, #1224, and #1225 need quite a bit of reworking. I don't understand the logic behind keeping 42 as a separate entry which just records all the notes/additions. The number of hands in 42 is inaccurate, because the latter ones are in fact main hands in 1224 and 1225. Then I wonder if the additions can be simply incorporated into the additions of 1224 and 1225?

**Comment by wlpotter** [comment]
Yes, this one is really complex...I'll go ahead and take over on it since it may require some major revisions to the XML.

We do have a way to nest an `msPart` inside another, so in this case we'd have Part 2.1 (ff. foll. 127—304) and Part 2.2 ( foll. 225—304). I'll take a closer look at this to make sure everything gets moved into the right section. Usually we use this for manuscripts that are palimpsest in only a portion of the manuscript. Here is somewhat interesting since Wright may be suggesting that this manuscript was two individual volumes rebound together, then later bound again with part 1 of this shelf mark.

In any case, I've assigned myself and can take care of the revisions on this one and for issues #711 and #710

**Comment by wlpotter** [comment]
Fixed
